src/lib/pickles/test/optional_custom_gates/pickles_test_optional_custom_gates.ml

```diff
@@ -239,6 +239,53 @@ let main_fixed_lookup_tables () =
        ; (* v3 *) w6 = exists Field.typ ~compute:(fun () -> v3)
        } )
 
+(* Parameters *)
+(* nb of fixed lookup tables *)
+let m = Random.State.int state 10
+
+let f_lt_data =
+  Array.init m ~f:(fun _ ->
+      let size = Random.State.int state 100 in
+      let indexes = Array.init size ~f:(fun i -> Field.Constant.of_int i) in
+      let values =
+        Array.init size ~f:(fun _ ->
+            Field.Constant.of_int (Random.State.int state 100_000_000) )
+      in
+      (indexes, values) )
+
+(* nb of lookups *)
+let n = Random.State.int state 10
```

Comment:
I assume the 0-10 questions might be the same here?

Reply:
See https://github.com/MinaProtocol/mina/pull/14056#discussion_r1319427471